src/docs/Help/Help/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Contact us'
+order: 1
+---
+
+> There are three main ways to get quick help from Mozaic. And they’re all pretty simple:
+
+## 1 - Via Slack.
+
+We have a dedicated slack channel inside [Adeo’s global Slack account](https://adeo-tech-community.slack.com). It’s called **#mozaic-support**. Usually, we take few minutes to answer. But depending on your timezone it might get a few hours on week-days.
+_There’s just one rule: Questions should be posted in English._
+
+## 2 - Via Github
+
+There’s another way, in case you don’t have access to slack, we can use [GitHub issues](https://github.com/adeo/mozaic-design-system/issues) as well. We might take a few hours to check that. Truth is, we’re not that used to receive GitHub issues. But as far as this gets popular, our response time will gain speed proportionally.
+_There’s just one rule: Questions should be posted in English._
+
+## 3 - Via e-mail
+
+Old but gold, e-mails never fail. But it’s true that it might take more time for us to answer via this channel. So we’d prefer to recommend you using either Slack or Github for proper and fast support.
+Currently, we support English, French, and Portuguese by e-mail 🙃 ! : tiago.rosa (at) adeo (dot) com
```

Comment:
~~tiago.rosa (at) adeo (dot) com~~ => I think we have to change the email adresse of Tiago :(